src/components/WorkflowForms/NewClientSignup/GatherInfo/hooks/useCaseData.js

```
@@ -0,0 +1,131 @@
+             reportNumber: currentReportNumber,
+             agency: currentAgency
+          };
+       } else if (fieldName === 'additionalPassengers') { 
+          propertiesToUpdate[fieldName] = value;
+       } else {
+          propertiesToUpdate[fieldName] = value;
+       }
+       
+       const result = await api.update_case_properties(caseId, propertiesToUpdate);
+       
+       if (result && result.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)